transport/http/outbound.go

```diff
@@ -86,7 +86,7 @@ func parseURL(urlStr string) (*url.URL, string) {
 // for getting potential downstream hosts.
 // PeerList.ChoosePeer MUST return *hostport.Peer objects.
 // PeerList.Start MUST be called before Outbound.Start
-func NewPeerListOutbound(peerList transport.PeerList, urlTemplate *url.URL) *Outbound {
+func NewPeerListOutbound(peerList peer.List, urlTemplate *url.URL) *Outbound {
```

Comment:
Nope, oneway implements a CallOneway (vs Call) function so the http.PeerListOutbound is compatible with both.